### ISSUE TITLE
Provide the logprobs or the logits

### ISSUE THREAD

**Issue by samartha-ramkumar** [ollama/ollama#10511]
If the model generates it anyway, why not provide it......It is very useful / necessary for certain downstream tasks. 
Most other providers provide it as part of the output.

**Comment by codelion** [comment]
@samartha-ramkumar you can try using optillm for it - https://github.com/codelion/optillm/issues/182
